osu.Game/Screens/Play/PlayerSettings/BeatmapOffsetControl.cs

```diff
@@ -230,10 +237,15 @@ private void scoreChanged(ValueChangedEvent<ScoreInfo?> score)
             }
 
             lastValidScore = score.NewValue!;
-            lastPlayMedian = median;
             lastPlayBeatmapOffset = Current.Value;
 
-            LinkFlowContainer globalOffsetText;
+            double unstableRate = hitEvents.CalculateUnstableRate()!.Result;
+
+            bool autoAdjustBeatmapOffset = config.Get<bool>(OsuSetting.AutomaticallyAdjustBeatmapOffset);
+
+            suggestedOffset = computeSuggestedOffset(median, unstableRate, lastPlayBeatmapOffset, proportionalToUnstableRate: autoAdjustBeatmapOffset);
```

Comment:
Can this method be inlined here again? It's only used once and extracting it out only seems to add complexity.

Reply:
Inlining the method here feels like shoving a lot of lines into this one single `scoreChanged` callback method, or at least that was what I had in mind. I'll inline it.